src/Sulu/Bundle/DocumentManagerBundle/Initializer/WorkspaceInitializer.php

```diff
@@ -39,13 +39,9 @@ public function initialize(OutputInterface $output)
             $workspace = $connection->getWorkspace();
 
             try {
-                $connection->getRootNode();
+                $workspace->createWorkspace($workspace->getName());
                 $output->writeln(sprintf('  [ ] <info>workspace</info>: "%s"', $workspace->getName()));
             } catch (RepositoryException $e) {
-                // TODO: We should catch the more explicit
-                // WorkspaceNotFoundException but Jackalope doctrine-dbal does
-                // not throw this: https://github.com/jackalope/jackalope-doctrine-dbal/issues/322
-                $workspace->createWorkspace($workspace->getName());
                 $output->writeln(sprintf('  [+] <info>workspace</info>: "%s"', $workspace->getName()));
```

Comment:
Some duplcation here .. basically if there was an RepositoryException then the workspace does NOT exist, so it will be `[+]` added, if no exception was thrown then it already exists (`[ ]` - no change).